controllers/front/SitemapController.php

```diff
@@ -91,7 +69,27 @@ public function initContent()
             true
         );
 
-        $this->context->smarty->assign('urls', $urls);
+        /*
+         * Backward compatibility with older themes.
+         * This should be removed as soon as possible, because $pages variable is overwriting
+         * our global template variable assigned in FrontController.
+         */
+        $this->context->smarty->assign(
```

Comment:
Can we get rid of this for 9.0, so we don't have to merge it ?

Reply:
I don't know what this is used for, but anyway it will not be removed in a merge PR